### ISSUE TITLE
Support for Paper servers.

### ISSUE THREAD

**Issue by Xantrisse** [mega12345mega/NBT-Editor#105]
Support for Paper servers allowing you to edit Inventory and Enderchest.

**Comment by BaiMeou** [comment]
Yes, I also really need a plugin version of this mods, which will make item editing easier
